mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price + price/5;
```

Comment:
It can be useful to declare separate variables at stages of each operation. When calculations become more complexed it may be necessary to check various stages to see if and when mistakes occur.